security/pfSense-pkg-HoneyTrap/files/usr/local/pkg/honeytrap/honeytrap-plugin.inc

```diff
@@ -0,0 +1,113 @@
+<?php
+/*
+ * honeytrap-plugin.inc
+ *
+ * part of pfSense (http://www.pfsense.org)
+ * Copyright (c) 2019 DutchSec (https://dutchsec.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require_once('globals.inc');
+require_once('pfsense-utils.inc');
+require_once('service-utils.inc');
+
+function honeytrap_sync_config() {
+    global $g, $config;
+    $gconfig = $config['installedpackages']['honeytrap']['config'][0];
```

Comment:
Initialize this config section with `init_config_arr()` before assign it to a variable